packages/nextjs/hooks/scaffold-eth/useAccountBalance.ts

```diff
@@ -19,8 +19,10 @@ export function useAccountBalance(address?: string) {
   });
 
   const onToggleBalance = useCallback(() => {
-    setIsEthBalance(!isEthBalance);
-  }, [isEthBalance]);
+    if (price > 0) {
```

Comment:
Can the price be negative? Should we just check for it to be a number? Would `if (typeof price === 'number') {` achieve the same?

Or is this to avoid division by 0?

Reply:
I don't think it has got anything to do with division by 0.

But what I think is if the currency price ever falls down to 0 or maybe even below 0 which I feel can only happen when the currency has lost its value or there is something wrong with uniswap SDK then probably it isn't right to display the price anyways. What do you think?